multi-node/health_check.py

```diff
@@ -300,8 +302,8 @@ def parse_nccl_output(raw_output, minBandwidthRequired, minAlgBandwidthRequired)
 OPERATION_ALL_REDUCE = 0
 OPERATION_ALL_TO_ALL = 1
 
-# These arguments are used for every nccl_tests command
-NCCL_ARGS = "-b8 -e16G -f2"
+# Default arguments are used for every nccl_tests command
+NCCL_ARGS = "-b1M -e1G -f2"
```

Comment:
But current arguments makes --min-alg-bandwidth and --min-bandwidth meaning less, because for small messages performance is mostly limited by latency, so it greatly affect final bandwidth numbers, so actual result will likely be (0 + max_bw) /2  but it is not guarantied. IMHO it is much more relevant to test real  use-cases which used by ML model.
It starts from megabytes, and ends with gigabytes, most models do not send huge tensors as single buffer, because it will block compute pipeline.
So if it is not too late let's change default values ranges from 1MB to 1GB, in that range bandwidth numbers does make sense, and it cover real usecases interested for end customers.

Reply:
This test isn't meant to be just a bandwidth test, or to imitate ML workloads. At this time, we would not like to change the default test sizes.

Reply:
@dbeer I've update PR and preserve original nccl-test options as you asked, but user is able to customize it.
Please review.